### DISCUSSION TITLE
Linking the API documentation in the repo URL

### DISCUSSION THREAD

**Discussion by janko** [libvips/ruby-vips#398] [General]
Hello, it would be convenient if the Ruby API documentation was linked on the repo itself, so that it appears below the repo description 🙂 I see there is no repo URL set at the moment.

**Comment by jcupitt** [comment]
Hi @janko, sorry, you'll need to be more specific. Do you mean in the gemspec metadata? We set `documentation_uri`.

**Reply by janko** [reply]
Ah, sorry, I meant here:

<img width="1440" alt="Screenshot 2024-06-06 at 11 46 15" src="https://github.com/libvips/ruby-vips/assets/795488/7938992c-8845-4879-9178-7cbf12781a3d">

There is a cog icon in that section that allows setting the website.
